website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -1,97 +1,62 @@
+## Installation
+
+### Prerequisites
 
-## Prerequisites
 - [git][git-tool]
-- [go][go-tool] version v1.13+
+- [go][go-tool] v1.13+
 
-## Download Operator SDK
+### Download Operator SDK
 
-Go to the [Operator SDK repo][repo-sdk] and follow the [fork guide][fork-guide] to fork, clone, and setup the local operator-sdk repository.
+Go to the [operator-sdk repo][repo-sdk] and follow the [fork guide][fork-guide] to fork and set up a local repository.
 
-## Build the Operator SDK CLI
+### Build the Operator SDK CLI
 
 Build the Operator SDK CLI `operator-sdk` binary:
 
 ```sh
-$ make install
+make install
 ```
 
-Then, now you are able to test and use the operator-sdk build using the source code.
-
 ## Testing
 
-The SDK includes many tests that are run as part of CI.
-To build the binary and run all tests (assuming you have a correctly configured environment),
-you can simple run:
-
-```sh
-$ make test-ci
-```
-
-If you simply want to run the unit tests, you can run:
-
-```sh
-$ make test
-```
-
-For more information on running testing and correctly configuring your environment,
-refer to the [`Running the Tests Locally`][running-the-tests] document. To incorporate code changes in your development environment see the [`Testing changes Locally`][testing-changes-locally] document.
-
-To run the lint checks done in the CI locally, run:
-
-```sh
-$ make lint
-```
-
-**NOTE** Note that for it is required to install `golangci-lint` locally. For more info see its [doc](https://github.com/golangci/golangci-lint#install)
-
-## How the operator-sdk binaries are built
-
-In the release process, the script [.release.sh][release-sh] is executed and it will call the [makefile][makefile] target [make release](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L113). To know more about the release process, see the [doc][release-doc] also see [operator-sdk releases](https://github.com/operator-framework/operator-sdk/releases).
-
-**NOTE** The Deploy stage (configured in [.travis.yml ][travis]) builds also execute the same [makefile][makefile] targets. This stage is executed against the master branch when a Pull Request is merged.
-
-## How to test the build of operator-sdk binaries
-
-Follow these steps to execute the Travis `Deploy` stage against your branch to demonstrate that the merge build will complete as expected.
+See the [testing][dev-testing] and [documentation][dev-docs] guides for more information.
 
-- Enable the Travis in your fork repository. For more info see [`To get started with Travis CI using GitHub`](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github) 
-- Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
```

Comment:
this info is important to have a doc. We need to have described the steps required that should be performed to tests changes related. E.g when we add a new image or a new arch type. Why we are removing it? Will be it moved for another place?